packages/snap/src/index.ts

```diff
@@ -94,6 +94,8 @@ export const onRpcRequest: OnRpcRequestHandler = async ({
         return ResultObject.success(res);
       case 'switchDIDMethod':
         isValidSwitchMethodRequest(request.params);
```

Comment:
Maybe update `isValidSwitchMethodRequest` to verify ebsiBearer is a 'valid' string.